geoh5py/objects/object_base.py

```diff
@@ -430,6 +428,20 @@ def find_or_create_property_group(
 
         return prop_group
 
+    def find_or_create_property_group(
+        self, name=None, uid=None, **kwargs
+    ) -> PropertyGroup:
+        """
+        Find or create :obj:`~geoh5py.groups.property_group.PropertyGroup`
+        from given name and properties.
+        """
+        warnings.warn(
+            "The 'find_and_create_property_group' will be deprecated. "
```

Comment:
haha, was just going to suggest this, then I refreshed to see you had read my mind!